src/main/scala/io/iohk/ethereum/faucet/FaucetHandler.scala

```diff
@@ -0,0 +1,102 @@
+package io.iohk.ethereum.faucet
+
+import akka.actor.{Actor, ActorLogging, Props}
+import akka.util.ByteString
+import io.iohk.ethereum.domain.Address
+import io.iohk.ethereum.faucet.FaucetHandler.WalletException
+import io.iohk.ethereum.faucet.FaucetStatus.WalletAvailable
+import io.iohk.ethereum.faucet.jsonrpc.WalletService
+import io.iohk.ethereum.keystore.KeyStore.KeyStoreError
+import io.iohk.ethereum.keystore.Wallet
+import monix.execution.Scheduler.Implicits.global
+
+class FaucetHandler(walletService: WalletService, config: FaucetConfig) extends Actor with ActorLogging {
+
+  import FaucetHandler.FaucetHandlerMsg._
+  import FaucetHandler.FaucetHandlerResponse._
+
+  var wallet: Wallet = _
```

Comment:
why not `context become available(wallet..)`?

Reply:
Done